addons/binding/org.openhab.binding.harmonyhub/ESH-INF/thing/thing-types.xml

```diff
@@ -1,31 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <thing:thing-descriptions bindingId="harmonyhub"
-		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
-		xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
 	<bridge-type id="hub">
 		<label>Harmony Hub</label>
 		<description>A Logitech Harmony Hub</description>
 		<channels>
-			<channel id="currentActivity" typeId="currentActivity" />
-			<channel id="activityStarting" typeId="eventTrigger">
-				<label>Activity Starting Trigger</label>
-				<description>Triggered when an activity is starting</description>
-			</channel>
-			<channel id="activityStarted" typeId="eventTrigger">
-				<label>Activity Started Trigger</label>
-				<description>Triggered when an activity is started</description>
-			</channel>
-		</channels>
+            <channel id="currentActivity" typeId="currentActivity" />
```

Comment:
Please use tabs everywhere in this file for indentation.

Reply:
👍 I had assumed eclipse did this automatically on save, by it did not :-(